### PR TITLE
Use Brim Suricata package (macOS, Ubuntu)

### PR DESCRIPTION
**Pull request by henridf**
This change adjusts our Makefile and a few related files to
automatically download and use the Brim suricata package when running
on macOS or Ubuntu. The process is overall similar to the one for zeek.

This also adjusts one system test to ensure that suricata ingest
completes properly.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,4 @@
-!bin/
-bin/*
-!bin/suricata/
+bin
```
Comment (nwt) [inline .gitignore:1]:
```suggestion
bin/
```

**Comment by henridf** [comment]
The linux package that's pulled from google cloud storage is built by https://github.com/brimsec/build-suricata/pull/2.
The macOS package is built manually at this point, automation to follow.

**Comment by henridf** [comment]
The markdown-link-check test failure is unrelated.

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -18,10 +18,8 @@ jobs:
       - run: make fmt
       - run: make tidy
       - run: |
-          sudo apt-get install -y software-properties-common
-          sudo add-apt-repository ppa:oisf/suricata-stable
           sudo apt-get update
-          sudo apt-get install -y suricata
+          sudo apt-get install -y zip libnet1 libjansson4
```
Comment (nwt) [inline .github/workflows/ci.yaml:22]: Can't this step go away?

Comment (henridf) [reply .github/workflows/ci.yaml:22]: No, the libraries are needed.

**Comment by henridf** [comment]
> The macOS package is built manually at this point, automation to follow.

Automation in https://github.com/brimsec/build-suricata/pull/3

File: .github/workflows/ci.yaml (hunk 2 above)
Comment (nwt) [reply .github/workflows/ci.yaml:22]: Won't that be a problem for Brim? (We go out of our way to link Zeek statically for that reason.)

Comment (henridf) [reply .github/workflows/ci.yaml:22]: As discussed elsewhere, I'd like use the regular .deb/.rpm dependency mechanism for this (just like the regular suricata .deb/.rpm does). If that doesn't work, then we can do the work to build statically. For now this does work for zq CI, which is what we need for the moment.

File: Makefile [hunk 3]
```diff
@@ -42,6 +43,13 @@ bin/$(ZEEKPATH):
 	@unzip -q bin/$(ZEEKPATH).zip -d bin \
 		&& mv bin/zeek bin/$(ZEEKPATH)
 
+bin/$(SURICATAPATH):
```
Comment (alfred-landrum) [inline Makefile:46]: Would you please `@` the lines below, as the `bin/$(ZEEKPATH)` target does, to remove the download status output?

**Review by alfred-landrum — COMMENTED** [review]
I pulled the branch to my mac, and hit two failures with make system-test:
TestZTest/suite/zqd/pcappost
TestZTest/suite/zqd/pcap-deleted

output here: https://gist.github.com/alfred-landrum/fe195bee18632bb46f3c26b84582c23d

**Comment by alfred-landrum** [comment]
If I try the running the commands for the pcap post test manually, zapi returns this error message:
```
/Users/alfred/work/zq/bin/suricata-v5.0.3-brim3/suricatarunner exited with status 1: 22/9/2020 -- 15:39:06 - <Error> - [ERRCODE: SC_ERR_MAGIC_LOAD(197)] - magic_load failed: could not find any valid magic files!
```

**Comment by henridf** [comment]
@alfred-landrum thanks for trying this out and looking into the issue. It should be fixed with https://github.com/brimsec/zq/pull/1331.

File: .gitignore (hunk 1 above)
Comment (mattnibs) [inline .gitignore:1]: Great to see this go!

**Review by mattnibs — APPROVED** [review]
`make-system` works on my machine

File: .gitignore (hunk 1 above)
Comment (henridf) [reply .gitignore:1]: 👍

File: tests/suite/zqd/pcappost.yaml [hunk 4]
```diff
@@ -26,3 +28,6 @@ outputs:
       1:[conn;1425567047.804914;[192.168.0.51;33773;80.239.174.91;443;]tcp;-;-;-;-;OTH;-;-;0;^d;0;0;1;1440;-;]
       1:[conn;1425567047.804906;[192.168.0.51;33773;80.239.174.91;443;]tcp;-;-;-;-;OTH;-;-;0;^d;0;0;1;1440;-;]
       1:[conn;1425567047.803929;[192.168.0.51;33773;80.239.174.91;443;]tcp;-;-;-;-;OTH;-;-;0;^d;0;0;1;1440;-;]
+      ===
+      #0:record[count:uint64]
+      0:[10;]
```
Comment (henridf) [inline tests/suite/zqd/pcappost.yaml:33]: without suricata this would be 9.

**Comment by alfred-landrum** [comment]
I removed `bin/suricata-v5.0.3-brim3*` from my checkout, verified that a new suricata pull happened (quietly), and then successfully ran `make test-system`.

**Review by alfred-landrum — APPROVED** [review]

**Pull request merged**